FYI looks like we have a few issues in the pipeline 👀  (These types seem to be referenced in another file)

```
❌  /Users/[REDACTED]/git/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageAction.swift:12:42: 'NetworkErrorType' is not a member type of class 'Client.NativeErrorPageHelper'

    let errorType: NativeErrorPageHelper.NetworkErrorType?
                   ~~~~~~~~~~~~~~~~~~~~~ ^

❌  /Users/[REDACTED]/git/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageAction.swift:17:42: 'NetworkErrorType' is not a member type of class 'Client.NativeErrorPageHelper'

        errorType: NativeErrorPageHelper.NetworkErrorType? = nil,
      ^

** TEST BUILD FAILED **

The following build commands failed:
	SwiftCompile normal arm64 /Users/[REDACTED]/git/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageAction.swift (in target 'Client' from project 'Client')
	Building project Client for testing with scheme Fennec
(3 failures)
```

--- a/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageAction.swift
+++ b/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageAction.swift
@@ -9,12 +9,10 @@ struct NativeErrorPageAction: Action {
     let windowUUID: WindowUUID
     let actionType: ActionType
     let networkError: NSError?
-    let errorType: NativeErrorPageHelper.NetworkErrorType?
     let nativePageErrorModel: ErrorPageModel?
 
     init(
         networkError: NSError? = nil,
-        errorType: NativeErrorPageHelper.NetworkErrorType? = nil,
         nativePageErrorModel: ErrorPageModel? = nil,
         windowUUID: WindowUUID,
         actionType: any ActionType
@@ -22,7 +20,6 @@ struct NativeErrorPageAction: Action {
         self.windowUUID = windowUUID
         self.actionType = actionType
         self.networkError = networkError
-        self.errorType = errorType
         self.nativePageErrorModel = nativePageErrorModel
     }
 }

--- a/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageHelper.swift
+++ b/firefox-ios/Client/Frontend/NativeErrorPage/NativeErrorPageHelper.swift
@@ -46,11 +46,6 @@ class NativeErrorPageHelper {
         let cert: SecCertificate
     }
 
-    enum NetworkErrorType {
-        case noInternetConnection
-        case badCertDomain
-    }
-
     var error: NSError
 
     var errorDescriptionItem: String {

